expense/expenseModel.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
```

Comment:
Abstracting the ObjectId type like this really comes in handy when using it in reference properties. This method is a great shorthand.